include/nautilus/percpu.h

```diff
@@ -218,7 +218,7 @@ get_cpu (void)
 
 #endif /* !__PER_CPU_H__ */
 
-#endif
+#endif /* NAUT_CONFIG_ARCH_RISCV */
```

Comment:
`percpu.h` is very fractured into arch-specific ifdef blocks, probably would work better as `arch/percpu.h` for later refactor